plugins/plugin-chart-echarts/src/Timeseries/transformers.ts

```diff
@@ -178,12 +178,14 @@ export function transformSeries(
     step: ['start', 'middle', 'end'].includes(seriesType as string) ? seriesType : undefined,
     stack: stackId,
     lineStyle,
-    areaStyle: {
-      opacity:
-        forecastSeries.type === ForecastSeriesEnum.ForecastUpper || area
-          ? opacity * areaOpacity
-          : 0,
-    },
+    areaStyle: area
+      ? {
+          opacity:
+            forecastSeries.type === ForecastSeriesEnum.ForecastUpper || area
```

Comment:
The conditional expression is always true. Maybe we can remove it here.

Reply:
Whoops, good catch!